### ISSUE TITLE
native directory not published for react-native

### ISSUE THREAD

**Issue by rdy** [facebookexperimental/Recoil#592]
When I installed the newly published 0.0.13 it looks like the `native/recoil.js` is not created, and it also does not have the `react-native` entry in `package.json` perhaps this due to some publishing issue?

I assumed this had to be due to some npm configuration but I didn't see anything obvious like an .npmignore, and the files seems to include the native directory.

**Comment by haswalt** [comment]
I’m interested where the 0.0.13 release came from since the version in master is currently 0.0.12 and there is a PR to update the package version currently open. https://github.com/facebookexperimental/Recoil/pull/581

**Comment by haswalt** [comment]
I’m not sure if 0.0.13 that was built is valid, I.e. was it built with the react-native changes in at all?

**Comment by rdy** [comment]
Yes, it is strange. It looks like master contains the changes for react-native, where it looks like 0.0.12 is the latest. The latest version I see on npmjs.com is 0.0.13, perhaps it is supposed to only have changes for SSR (according to the changelogs PR)?

Might want to see what ended up on npmjs.com just to make sure.

**Comment by mondaychen** [comment]
React native support was not published with 0.0.13. It was published before the PR was merged.

**Comment by mondaychen** [comment]
Hi everyone! FYI we've published a "nightly build" branch for testing purposes. If you want to try out recoil with react native before we release next version, try install the `nightly` branch with the "install git branch" feature of npm/yarn:

```
npm install https://github.com/facebookexperimental/Recoil.git#nightly
```
Or
```
yarn add https://github.com/facebookexperimental/Recoil.git#nightly
```
